pkgs/utils/src/log.zig

```diff
@@ -4,8 +4,8 @@ const datetime = @import("datetime");
 
 // having activeLevel non comptime and dynamic allows us env based logging and even a keystroke activated one
 // on a running client, may be can be revised later
-pub fn compTimeLog(comptime scope: LoggerScope, activeLevel: std.log.Level, comptime level: std.log.Level, comptime fmt: []const u8, args: anytype) void {
-    if (@intFromEnum(level) > @intFromEnum(activeLevel)) {
+pub fn compTimeLog(comptime scope: LoggerScope, activeLevel: std.log.Level, comptime level: std.log.Level, comptime fmt: []const u8, args: anytype, fileParams: ?FileParams) void {
```

Comment:
don't pass full fileParams but cast it down {fileActiveLevel, file} only so declare another struct and use that here,